.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+    "configurations": [
+        {
+            "name": "Dynamic Python: File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "justMyCode": false,
+            "env": {
+                "OFFLINE_TOKEN": "YOUR OFFLINE TOKEN HERE"
```

Comment:
Process would also have access to system env. Thus this is going to override it and be generally less useful than no env defined
```suggestion
```